.gitignore

```diff
@@ -30,3 +30,7 @@ bindings
 # Database cluster and sockets for testing
 dev_pgdata/
 *.PGSQL.*
+
+# Devenv
```

Comment:
Change this to nixos because I have no idea what it means.

Reply:
Done